admin_guide/diagnostics_tool.adoc

```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
 `system_u:object_r:container_file_t:s0:c113,c247`. For more details about `:Z`,
 see the `docker-run(1)` man page.
 
-Keep this in mind for these volume mount specifications ￼because it could have
-unexpected consequences. For example, if you mount (and ￼therefore relabel) your
-*_$HOME/.ssh_* directory, *sshd* will become unable to access your public keys
-to allow remote login. To avoid altering the original file labels, mounting a
-copy of the SSH key (or directory) is recommended.
+[IMPORTANT]
+====
+These volume mount specifications could have unexpected consequences. For
+example, if you mount (and ￼therefore relabel) the *_$HOME/.ssh_* directory,
+*sshd* becomes unable to access the public keys to allow remote login. To avoid
+altering the original file labels, mounting a copy of the SSH key (or directory)
+is recommended.
+====
 
-You might want to mount an entire *_.ssh_* directory for various reasons. For
-example, this would allow you to use an SSH configuration to match keys with
-hosts or modify other connection parameters. It would also allow you to provide
-a *_known_hosts_* file and have SSH validate host keys, which is disabled by the
+You might mount an entire *_.ssh_* directory for various reasons. For example,
```

Comment:
I think this is part of the previous note.

Despite the risks of mounting an entire *_.ssh_* directory, mounting the directory offers benefits:
* You can use an SSH configuration to match keys with hosts or modify other connection parameters.
* You can provide a *_known_hosts_* file and have SSH validate host keys. This behavior is disabled by the
 default configuration. To use it, add `-e ANSIBLE_HOST_KEY_CHECKING=True` to the `docker run` command.